xsd/netex_part_3/part3_fares/netex_accessRightParameter_version.xsd

```diff
@@ -412,12 +416,6 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>ORGANISATION validity parameters for ACCESS RIGHT PARAMETER ASSIGNMENT.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:element name="VehicleModes" type="VehicleModeListOfEnumerations" minOccurs="0">
```

Comment:
Why in this one removed?

Reply:
This file is for ACCESS RIGHT for FARES, and the this  OrganisationValidityParametersGroup is to connect them to ORGANISATION, not to MODEs, there is a ModeValidityParametersGroup for this. So this looks Ok (was here by mistake I guess)